mono_repo.yaml

```diff
@@ -10,7 +10,3 @@ github:
       branches: master
     schedule:
       - cron: "0 0 * * 0"
-
-merge_stages:
```

Comment:
Fwiw, I would keep this, and just list the new stage name.

Generally merging saves on CI resources by merging jobs with the same SDK/OS together, reducing the overhead for the setup of those jobs, which is usually most of the work.

Reply:
I brought this back, but it didn't cause any changes in the GitHub CI files. Did I do it wrong?

Reply:
Maybe I was wrong about how this actually works.... lol.

Reply:
I removed this again as it doesn't seem to make any difference.